code-challenges.js

```diff
@@ -39,16 +51,55 @@ var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 
 var vowelTester1 = "learn"
 // Expected output: 1
-var vowelTester1 = "throw"
+var newVowelTester1 = vowelTester1.split("")
+var vowels = ["a","e","i","o","u"]
+
+var vowelTester2 = "throw"
 // Expected output: 3
+var newVowelTester2 = vowelTester2.split("")
+
+function findCommonElement1(array1, array2) {
 
+    // Loop for array1
+    for(let i = 0; i < array1.length; i++) {
 
+        // Loop for array2
+        for(let j = 0; j < array2.length; j++) {
 
+            // Compare the element of each and
+            // every element from both of the
+            // arrays
+            if(array1[i] == array2[j]) {
+                return i;
+            }
+        }
+    }
+    return false;
+}
```

Comment:
good job! as an added challenge, think about how you might be able to incorporate the `.split()` for your `vowelTester` variables without having to create brand new variables outside of the function.